pepper-apis/src/main/java/org/broadinstitute/ddp/model/statistics/StatisticsType.java

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.ddp.model.statistics;
+
+import java.beans.ConstructorProperties;
+
+public class StatisticsType {
```

Comment:
I recommend removing this class and just use the `enum`.

Reply:
Removed